inference-engine/tests/functional/inference_engine/local_test.cpp

```diff
@@ -15,183 +15,589 @@ using namespace InferenceEngine;
-                    <dim>1</dim>
-                    <dim>30</dim>
-                </port>
-            </output>
-        </layer>
-        <layer id="1" name="Split" precision="FP32" type="Split">
-            <data axis="1" />
-            <input>
-                <port id="0">
-                    <dim>1</dim>
-                    <dim>30</dim>
-                </port>
-            </input>
-            <output>
-                <port id="1">
-                    <dim>1</dim>
-                    <dim>10</dim>
-                </port>
-                <port id="2">
-                    <dim>1</dim>
-                    <dim>10</dim>
-                </port>
-                <port id="3">
-                    <dim>1</dim>
-                    <dim>10</dim>
-                </port>
-            </output>
-        </layer>
-        <layer id="2" name="LSTMCell" precision="FP32" type="LSTMCell">
-            <data hidden_size="10" clip="0.2"/>
-            <input>
-                <port id="0">
-                    <dim>1</dim>
-                    <dim>10</dim>
-                </port>
-                <port id="1">
-                    <dim>1</dim>
-                    <dim>10</dim>
-                </port>
-                <port id="2">
-                    <dim>1</dim>
-                    <dim>10</dim>
-                </port>
-            </input>
-            <output>
-                <port id="3">
-                    <dim>1</dim>
-                    <dim>10</dim>
-                </port>
-                <port id="4">
-                    <dim>1</dim>
-                    <dim>10</dim>
-                </port>
-            </output>
-            <blobs>
-                <weights offset="0" size="3200"/>
-                <biases offset="3200" size="160"/>
-            </blobs>
-        </layer>
-        <layer name="Eltwise" type="Eltwise" id="3" precision="FP32">
-            <data operation="sum" />
-            <input>
-                <port id="0">
-                    <dim>1</dim>
-                    <dim>10</dim>
-                </port>
-                <port id="1">
-                    <dim>1</dim>
-                    <dim>10</dim>
-                </port>
-            </input>
-            <output>
-                <port id="2">
-                    <dim>1</dim>
-                    <dim>10</dim>
-                </port>
-            </output>
-        </layer>
-        </layers>
-        <edges>
-            <edge from-layer="0" from-port="0" to-layer="1" to-port="0"/>
-            <edge from-layer="1" from-port="1" to-layer="2" to-port="0"/>
-            <edge from-layer="1" from-port="2" to-layer="2" to-port="1"/>
-            <edge from-layer="1" from-port="3" to-layer="2" to-port="2"/>
-            <edge from-layer="2" from-port="3" to-layer="3" to-port="0"/>
-            <edge from-layer="2" from-port="4" to-layer="3" to-port="1"/>
-        </edges>
-    </net>
+<net name="model" version="10">
+	<layers>
```

Comment:
I don't see floating point numbers in LSTM model

Reply:
added